src/Sentry.Extensions.Logging/SentryLogger.cs

```diff
@@ -99,19 +99,20 @@ public bool IsEnabled(LogLevel logLevel)
             if (ShouldAddBreadcrumb(logLevel, eventId, exception))
             {
                 var data = eventId.ToDictionaryOrNull();
+
                 if (exception != null && message != null)
                 {
                     // Exception.Message won't be used as Breadcrumb message
                     // Avoid losing it by adding as data:
-                    data = data ?? new Dictionary<string, string>();
+                    data ??= new Dictionary<string, string>();
                     data.Add("exception_message", exception.Message);
                 }
 
                 _hub.AddBreadcrumb(
                     _clock,
-                    message ?? exception?.Message,
+                    message ?? exception?.Message!,
```

Comment:
@bruno-garcia note the `!` here. This is needed because we annotated `message` as non-nullable even though it technically is, in fact nullable.

Reply:
OK, breadcrumb without a message looks crappy (useless?) but it's supported.
Let's leave the non-nullable annotation